Userland/Libraries/LibGL/Tex/Sampler2D.cpp

```diff
@@ -21,12 +21,12 @@ static constexpr float wrap_mirrored_repeat(float value)
     float integer = floorf(value);
     float frac = value - integer;
     bool iseven = fmodf(integer, 2.0f) == 0.0f;
-    return iseven ? frac : 1 - frac;
+    return iseven ? frac : 1 - frac - NumericLimits<float>::epsilon();
```

Comment:
See my other comment on clamping

Reply:
We use `GL_CLAMP_TO_EDGE` behaviour for now.